README.md

```diff
@@ -1,3 +1,4 @@
 # TicTacToe_Ruby_v2
 
-[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=master)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
+
+[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=convert_to_gem)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
```

Comment:
If this gets merged into master, then the master branch will be showing the build status of the `convert_to_gem` branch -- correct?